components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.ts

```diff
@@ -30,21 +34,30 @@ export class ServicesSidebarComponent implements OnInit {
 
     this.serviceGroupName$ = this.store.select(createSelector(serviceGroupState,
       (state) => state.selectedServiceGroupName));
+
+    this.currentPage = 1;
   }
 
   public closeServicesSidebar() {
     this.closeServicesSidebarEvent.emit(null);
   }
 
-  public updateServicesFilters(health: string): void {
+  public updateServicesFilters(health: string, page?: number): void {
```

Comment:
Is the page an optional parameter? (because you have a `?` at the end) I don't see any case where we don't send a page number.

Reply:
If you call updateServiceFilters from the page filter bars you don't send the page number.

Reply:
(from the HTML code, yeah!) That makes sense! Then leave it as optional but then my other comment applies, lets use `this.currentPage` instead of `page` or, do we expect the "status filter bar" to reset the page to `1`?